content/blog/2018/egress-https/index.md

```diff
@@ -2,40 +2,60 @@
 title: Consuming External Web Services
 description: Describes a simple scenario based on Istio's Bookinfo example.
 publishdate: 2018-01-31
-subtitle: Egress Rules for HTTPS traffic
+subtitle: Mesh-external service entries for egress HTTPS traffic
 attribution: Vadim Eisenberg
 weight: 93
 keywords: [traffic-management,egress,https]
 ---
 
-In many cases, not all the parts of a microservices-based application reside in a _service mesh_. Sometimes, the microservices-based applications use functionality provided by legacy systems that reside outside the mesh. We may want to migrate these systems to the service mesh gradually. Until these systems are migrated, they must be accessed by the applications inside the mesh. In other cases, the applications use web services provided by external organizations, often over the World Wide Web.
+> This blog post was updated on August 09, 2018. It reflects the functionality of Istio 1.0 and uses the new
+[v1alpha3 traffic management API](/blog/2018/v1alpha3-routing/). If you need to use the old version, follow the docs
+[here](https://archive.istio.io/v0.7/blog/2018/egress-https.html).
 
-In this blog post, I modify the [Istio Bookinfo Sample Application](/docs/examples/bookinfo/) to fetch book details from an external web service ([Google Books APIs](https://developers.google.com/books/docs/v1/getting_started)). I show how to enable external HTTPS traffic in Istio by using an _egress rule_. Finally, I explain the current issues related to the egress traffic control in Istio.
+In many cases, not all the parts of a microservices-based application reside in a _service mesh_. Sometimes, the
+microservices-based applications use functionality provided by legacy systems that reside outside the mesh. You may want
+to migrate these systems to the service mesh gradually. Until these systems are migrated, they must be accessed by the
+applications inside the mesh. In other cases, the applications use web services provided by third parties, often over
+the World Wide Web.
 
-## Bookinfo sample application with external details web service
+In this blog post, I modify the [Istio Bookinfo Sample Application](/docs/examples/bookinfo/) to fetch book details from
```

Comment:
Rather than the `I`s, why not adopt the `we` ? In this blog post, we will see how to ..., we will configure ..., etc.

Reply:
@rshriram Note that the style guide recommends avoiding "we" https://istio.io/about/contribute/style-guide/#address-the-reader-as-you